deploy/examples/operator-openshift.yaml

```diff
@@ -127,10 +127,14 @@ data:
   # there is significant drop in read/write performance.
   # CSI_ENABLE_HOST_NETWORK: "true"
 
-  # Set logging level for csi containers.
+  # Set logging level for Kubernetes-csi sidecar containers.
   # Supported values from 0 to 5. 0 for general useful logs, 5 for trace level verbosity.
   # CSI_LOG_LEVEL: "0"
 
+  # Set logging level for cephCSI containers maintained by the cephCSI.
```

Comment:
Is this comment switched with line 130? This description is reversed from my understanding.

Reply:
ahh yeah true, Thanks. Updating now.